maintainers/maintainer-list.nix

```
@@ -8940,6 +8940,12 @@
     githubId = 13622947;
     keys = [ { fingerprint = "1412 816B A9FA F62F D051 1975 D3E1 B013 B463 1293"; } ];
   };
+  istoph = {
+    email = "chr@istoph.de";
+    name = "Christoph Hüffelmann";
+    github = "istoph";
+    githubID = 114227790;
```

```suggestion
    githubId = 114227790;
```

Obligatory typo right before finishing.